roles/reproducer/tasks/libvirt_layout.yml

```diff
@@ -91,7 +91,7 @@
 # match the multi-hypervisor schema.
 - name: Configure compute network
   when:
-    - compute.key is match('^compute.*')
+    - "compute.key in groups['computes'] or compute.key in (groups['cephs'] | default([]))"
```

Comment:
my only issue here is about the way to match the hosts - there's an assumption of names being, really "compute" and "ceph" in the libvirt_manager_configuration.vms
For now, it's OK, because we kind of know and agree on that. But maybe we should get a better way - later, once we hit actual issue.